sonar-db/src/main/java/org/sonar/db/qualitygate/ProjectQgateAssociationQuery.java

```diff
@@ -43,7 +45,7 @@
   private final String projectSearch;
 
   // for internal use in MyBatis
-  private final String projectSearchSql;
+  private final String projectSearchUpperLikeSql;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused "projectSearchUpperLikeSql" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)